app/assets/javascripts/remote_consoles/spice.js

```diff
@@ -13,7 +13,7 @@ $(function() {
 
   $('#ctrlaltdel').click(sendCtrlAltDel);
 
-  var spice = new SpiceMainConn({
+  sc = new SpiceMainConn({
```

Comment:
If this is really necessary, I would name it something more descriptive as the `sc` can mean anything and mistakenly can be overridden by something else stored as `sc`. What was the problem with the original name?

Reply:
Also a few lines below there's a reference to the original `spice`, so the `sc` naming breaks it.

Reply:
The problem with the name is that is what is referred to throughout the Spice JS code, so our hands are tied a bit there until it is updated to stop polluting the global namespace. 

Originally declared:
https://github.com/himdel/spice-html5-bower/blob/master/spiceHTML5/spice_auto.html#L138

Reference specific to ctrl-alt-del function:
https://github.com/himdel/spice-html5-bower/blob/master/spiceHTML5/inputs.js#L189

Reply:
@himdel what do you think?

Reply:
Well, `spice_auto` is technically creating a global named `sc`, and assigning a new `SpiceMainConn` instance there on `connect`.

`connect` is being called from the file, which means that maybe `sc` is the right variable to work with.

**But** if that's the case, creating a new `SpiceMainConn` instance we don't ever use doesn't make sense and we need to remove that call completely.

Reply:
@himdel @skateman This should be consistent with ```spice_auto``` now with a global declaration and initializing it later.

Reply:
I'm starting to get confused... :confused:

Reply:
Looking :).. so far, looks like `spice_auto.html` is only being used in sui, not in ops.

But.. that `sendCtrlAltDel` is a global function (defined in spice-html5) which uses a global `sc` variable - so that's the issue.

So... a good fix would be to update that `sendCtrlAltDel` function to actually get the right instance as a parameter, and then we wouldn't have to do this.

But, if we don't want to change the library internals, this makes perfect sense now :).